ClinGen/clincoded
# Connects to #1505. Connects to #1509. Connects to #1510.

**Technical notes:**
1. There are no testing steps for #1509. Refer to the ticket for details.
2. There are no testing steps for #1510. Refer to the ticket for details.
3. Updated BDD test assertions on OLS disease lookup and Indel variant in VCI.

**Steps to test #1505:**
1. In the VCI, select an **"Indel"** variant (e.g. **CA ID** _CA645372267_ or **ClinVar ID** _231244, 423947_) and then proceed to the Evidence-Only view of the VCI.
2. On the **"Basic Information"** tab, expect to see the correct transcripts in the **"RefSeq"** and **"Ensembl"** transcripts tables.
3. Then try selecting another **"Indel"** variant (e.g. **ClinVar ID** _53981_) and expect to see no **"RefSeq"** or **"Ensembl"** transcripts being returned. Instead, there would be only messages shown in the table panels, such as "Unable to return RefSeq transcripts for this allele. Search RefSeq for transcripts" or "Unable to return Ensembl transcripts for this allele. Search Ensembl for transcripts."
